server/chat-commands/moderation.ts

```diff
@@ -1537,9 +1537,12 @@ export const commands: ChatCommands = {
 	],
 
 	fr: 'forcerename',
-	forcerename(target, room, user) {
+	frinstaff: 'forcerename',
+	forcerenameinstaff: 'forcerename',
+	forcerename(target, room, user, connection, cmd) {
 		if (!target) return this.parse('/help forcerename');
 
+		if (cmd.endsWith('staff') && Rooms.search('staff')) room = this.room = Rooms.search('staff')!;
```

Comment:
```suggestion
	forcerename(target, room, user, connection, cmd) {
		if (!target) return this.parse('/help forcerename');

```

Consensus seems to be no need for `/frinstaff`.